### PR TITLE
Update release notes draft to version v1.19.0-beta.1

### PR DESCRIPTION
**Pull request by puerco**
**What type of PR is this?** /kind documentation **What this PR does / why we need it**: This PR updates the release notes draft to k/k v1.19.0-beta.1 **Which issue(s) this PR fixes**: **Special notes for your reviewer**: This is an automated PR generated from `krel The Kubernetes Release Toolbox`

### CODE REVIEW

**Comment by puerco** [comment]
This is the first PR to update to the release notes draft generated automatically. The PR body is missing the end of lines, other than that I think it looks fine. wdyt @saschagrunert ?

File: releases/release-1.19/release-notes-draft.md [hunk 1]
```diff
@@ -195,22 +231,29 @@
...
 - Kubeadm: do not use a DaemonSet for the pre-pull of control-plane images during "kubeadm upgrade apply". Individual node upgrades now pull the required images using a preflight check. The flag "--image-pull-timeout" for "kubeadm upgrade apply" is now deprecated and will be removed in a future release following a GA deprecation policy. ([#90788](https://github.com/kubernetes/kubernetes/pull/90788), [@xlgao-zju](https://github.com/xlgao-zju)) [SIG Cluster Lifecycle]
 - Kubeadm: use two separate checks on /livez and /readyz for the kube-apiserver static Pod instead of using /healthz ([#90970](https://github.com/kubernetes/kubernetes/pull/90970), [@johscheuer](https://github.com/johscheuer)) [SIG Cluster Lifecycle]
-- Node ([#76443](https://github.com/kubernetes/kubernetes/pull/76443), [@mgdevstack](https://github.com/mgdevstack)) [SIG Architecture, Network, Node, Testing and Windows]
+- NONE ([#91597](https://github.com/kubernetes/kubernetes/pull/91597), [@elmiko](https://github.com/elmiko)) [SIG Autoscaling and Testing]
```
Comment (saschagrunert) [inline releases/release-1.19/release-notes-draft.md:246]: This looks suspicious 🙂

**Comment by AGMETEOR** [comment]
Thanks @puerco 
/lgtm

**Pull request merged**